tests/define_group_test_project.py

```diff
@@ -0,0 +1,87 @@
+import flow
```

Comment:
Please make sure to add at least one test where the group-decorator is not the one on top (see comment earlier).